README.md

```diff
@@ -139,6 +139,10 @@ cfnLint.validateFile(fileName: string, options?: ValidationOptions): ValidationR
 ```
 Validates a file, and returns an ValidationResult with the results.
 
+```ts
+cfnLint.validateString(contents: string, fileName: string, options?: ValidationOptions): ValidationResult
```

Comment:
Why would a path be required if we are going to be taking in a raw string?

Reply:
The path is used by the YAML parser for error reporting:

https://github.com/martysweet/cfn-lint/blob/dbbe62d4bac20915d73c1f7d33440c4947db7e6b/src/parser.ts#L40-L41

Reply:
Is it required? Could we pass in a mock value, like "stdin"?

Reply:
> "filename (default: null) - string to be used as a file path in error/warning messages."

I would suggest we do not require a filename on the `validateString` API and instead pass `null` into the yaml parser instead when called from this method.